waku/v2/node/jsonrpc/relay_api.nim

```diff
@@ -2,12 +2,21 @@ import
   json_rpc/rpcserver,
   eth/[common, rlp, keys, p2p],
   ../../waku_types,  
-  ../wakunode2
+  ../wakunode2,
+  ./jsonrpc_types, ./jsonrpc_utils
 
 proc installRelayApiHandlers*(node: WakuNode, rpcsrv: RpcServer) =
   const futTimeout = 5.seconds
 
   ## Relay API version 1 definitions
+  
+  rpcsrv.rpc("post_waku_v2_relay_v1_message") do(topic: string, message: WakuRelayMessage) -> bool:
+    ## Publishes a WakuMessage to a PubSub topic
+    debug "post_waku_v2_relay_v1_message"
+
+    node.publish(topic, message.toWakuMessage(version = 0))
```

Comment:
Maybe await here and return in case of errors? Though that's not how publish is setup right now, so slightly out of scope of this PR. Relates to https://github.com/status-im/nim-waku/issues/146

Reply:
Yes, this will certainly be an improvement. Will address as a separate subsequent task.